src/components/SigninForm/index.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import {
```

Comment:
hey Herman, do you prefer the vertical approach or just inline? 💭

Reply:
I prefer the vertical approach only if there are more than 5 items to import.

Reply:
awesome, I'll keep that in mind